tests/unit/models/blueprint-test.js

```diff
@@ -285,8 +285,8 @@ describe('Blueprint', function() {
         });
     });
 
-    it('re-installing conflicting files', function() {
-      return blueprint.install(options)
+    it('re-installing conflicting files', function(done) {
+      blueprint.install(options)
```

Comment:
We should figure out why and fix it, without this we will have annoyingly brittle tests